todo-app/src/ToDo.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import TodoDesc from './TodoDesc.js'
+import ToDoStatus from './ToDoStatus.js'
+import './todo.css'
+
+
+export default class TODO extends React.Component {
+    render() {
+        const { todoDesc, toDoStatus } = this.props.toDo
+        return (
+            <div className="toDoItem">
+                <div>
```

Comment:
You didn't really need the extra `div`s around the components. This should work as well

```jsx
<div className="toDoItem">
  <TodoDesc todoDesc={todoDesc} />
  <ToDoStatus toDoStatus={toDoStatus} />
</div>
```